apps/frontend/src/libs/components/sidebar/styles.module.css

```
@@ -0,0 +1,27 @@
+.sidebar {
+	position: fixed;
+	top: 0;
+	left: 0;
+	z-index: 1;
+	display: flex;
+	flex-direction: column;
+	gap: 56px;
+	width: 360px;
```

It's 320 on the design and don't forget to avoid fixed values
```suggestion
	max-width: 320px;
```